Sources/SnapshotTestCase.swift

```diff
@@ -68,91 +69,120 @@ open class SnapshotTestCase: XCTestCase {
 
         try self.assert(texture: elementTexture,
                         testName: testName,
-                        threshold: threshold,
-                        recording: recording)
+                        options: options)
     }
 
     public func assert(screenshot: XCUIScreenshot,
                        testName: String,
                        ignoreStatusBar: Bool = true,
-                       threshold: Float = 10,
-                       recording: Bool = false) throws {
-        let texture: MTLTexture
-
+                       options: SnapshotOptions) throws {
         guard let cgImage = screenshot.image.cgImage
         else { throw Error.snaphotAssertingFailed }
         let screenTexture = try self.context.texture(from: cgImage, srgb: false)
 
         if ignoreStatusBar {
             var yOffset: CGFloat = 0
-            if !self.device.isOneOf(Device.allDevicesWithSensorHousing) {
+            let sensorHousing = Device.allDevicesWithSensorHousing + Device.allSimulatorDevicesWithSensorHousing
```

Comment:
I personally prefer operator alignment

```suggestion
let sensorHousing = Device.allDevicesWithSensorHousing
                  + Device.allSimulatorDevicesWithSensorHousing
```